### ISSUE TITLE
Icon for Get Started App not visible

### ISSUE THREAD

**Issue by kemppaik** [msft-teams/tools#10]
I have uploaded the Get Started app using the instructions and in our Teams there is no icon appear - it is just a greyed out space.  The app appears to work once it appears for everyone, but the colored icon does not appear:

What the icon appears as in the Admin center:
![image](https://user-images.githubusercontent.com/55152539/83184449-b94d6200-a0e6-11ea-9779-318ed982bd71.png)

What users see when opening Apps from within Teams
![image](https://user-images.githubusercontent.com/55152539/83184569-e00b9880-a0e6-11ea-8f3d-ba98809b294e.png)

Any suggestions/thoughts as to why the icon is not appearing?

**Comment by J-Krepps-OH** [comment]
This has been an intermittent issue for us as well, though most of the time signing out of Teams and back in resolves it. I look forward to seeing if you get any resolution/root cause.

**Comment by karuanag** [comment]
@kemppaik Please let us know if this issue is resolved. Thank you.

**Comment by Marshgate** [comment]
issue is perm for me. using the following versions:
Using versions below:
 "manifestVersion": "1.5",
    "version": "1.6.3",

**Comment by kemppaik** [comment]
@karuanag :  No, this issue is not resolved.  Like J-Kreeps-OH reported, it is an intermittent issue for us now.  Most of the time signing in of Teams and back in resolves it, but not a 100% guarantee.

**Comment by jasonwbaxter** [comment]
@karuanag  could it be related to the icon sizes , Teams wants 192X192 and 32x32 we are supplying 400px size icons. I've resized these in paint , and my icon loads no problem when i deploy the app.
